opencensus/metrics/export/measurement.py

```diff
@@ -0,0 +1,53 @@
+# Copyright 2019, OpenCensus Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import time
+
+from opencensus.ext.azure.common import utils
+
+class Measurement(object):
+    """ A measurement is an object with a value and timestamp attached to it
+        It represents a single raw data point
+        The timestamp is set to the current time that the Measurement is created
+
+    :type value: long or double
+    :param value: value of the measurement
+
```

Comment:
There are some formatting problems here -- too-long lines, extra spaces, leading indentation. Lint seems not to catch these though.